cmd/ingress-operator/main.go

```diff
@@ -55,9 +55,10 @@ func main() {
 	// Collect operator configuration.
 	operatorNamespace := os.Getenv("WATCH_NAMESPACE")
 	if len(operatorNamespace) == 0 {
-		log.Error(fmt.Errorf("missing environment variable"), "'WATCH_NAMESPACE' environment variable must be set")
-		os.Exit(1)
+		operatorNamespace = "openshift-ingress-operator"
```

Comment:
This looks like a leftover from development that should be reverted.

Reply:
It was certainly useful — why not keep it?

Reply:
Acceptable.